ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -118,49 +125,123 @@ ACP-103 introduced the following variables for calculating the gas price:
 
 </div>
 
-ACP-176 set $R = 2T$ and provided a mechanism to make $T$ dynamic.
+ACP-176 provided a mechanism to make $T$ dynamic and set:
+
+$$
+\begin{align}
+R &= 2 \cdot T \\
+K &= 87 \cdot T
+\end{align}
+$$
+
 The _excess_ actual consumption $x \ge 0$ beyond the target $T$ is tracked via numerical integration and used to calculate the gas price as:
 
 $$M \cdot \exp\left(\frac{x}{K}\right)$$
 
-While ACPs 103 and 176 compute this price at the resolution of a single block, we can instead integrate at the resolution of a single transaction with equivalent pricing dynamics.
-Exploiting the proportionality of $R = pT$ with $p \ge 1$, for a transaction consuming $g$ gas, the excess is increased:
+### Gas charged
+
+We introduce $g_L$, $g_U$, and $g_C$ as the gas _limit_, _used_, and _charged_ per transaction, respectively. We define
+
+$$
+g_C := \max\left(g_U, \frac{g_L}{\lambda}\right)
+$$
+
+where $\lambda$ enforces a lower bound on the gas charged based on the gas limit.
+
+> [!NOTE]
+> $\dfrac{g_L}{\lambda}$ is rounded up by actually calculating $\dfrac{g_L + \lambda - 1}{\lambda}$
+
+In all cases that execution used to refer to gas usage, we will from now on consider gas charged. For example, the gas excess $x$ will be modified by $g_C$ rather than $g_U$.
+
+### Queue size
+
+The constant time delay between execution and settlement is defined as $\tau$ seconds.
+
+The maximum allowed size of the execution queue is defined as:
+
+$$
+\omega ~:= R \cdot \tau \cdot \lambda
+$$
+
+Any block that could cause the total sum of gas limits for transactions in the execution queue to exceed $\omega$ MUST be considered invalid.
```

Comment:
```suggestion
Any block that would cause the total sum of gas limits for transactions in the execution queue to exceed $\omega$ MUST be considered invalid.
```

Reply:
I used "could" here to try to convey that this check isn't referring to the local state of the executor. But instead the global worst-case execution assumption.